airflow/providers/alibaba/cloud/hooks/oss.py

```diff
@@ -269,6 +279,74 @@ def create_bucket(
             self.log.error(e)
             raise AirflowException(f"Errors when create bucket: {bucket_name}")
 
+    @provide_bucket_name
+    @unify_bucket_name_and_key
+    def append_string(self, bucket_name: Optional[str], content: str, key: str, pos: int):
+        """
+        Append string to a remote existing file
+
+        :param bucket_name: the name of the bucket
+        :param content: content to be appended
+        :param key: oss bucket key
+        :param pos: position of the existing file where the content will be appended
+        """
+        self.log.info("Write oss bucket key: " + key)
+        self.log.info("Write oss bucket pos: " + str(pos))
```

Comment:
```suggestion
        self.log.info("Write oss bucket. key: %s, pos: %s", key, pos)
```
Pass the text and arguments to the logger separately so that they can be formatted depending on the logger configuration. See: https://github.com/apache/airflow/blob/8505d2f0a4524313e3eff7a4f16b9a9439c7a79f/airflow/utils/log/colored_log.py#L39

Reply:
Thx for the suggestions, will fix it in the following commit.